presto-main/src/main/java/com/facebook/presto/sql/planner/PlanPrinter.java

```diff
@@ -544,6 +562,25 @@ private void printDistributions(int indent, PlanNodeStats nodeStats)
         return ImmutableMap.of();
     }
 
+    private void printWindowNodeStats(int indent, WindowNodeStats stats)
+    {
+        output.append(indentString(indent));
+        output.append(format("Active Drivers: [ %d / %d ]", stats.getActiveDrivers(), stats.getTotalDrivers()));
```

Comment:
This is mostly for us because debugging WFs without understanding how they are executed (pipeline vs indexed approach etc.) won't be efficient.
The number of active drivers would tell us about the skew and concurrency in WF execution.